src/constants.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+DATA_PATH = os.path.join("..", "src", "data", "list_attr_celeba.csv")
+DATASET_PATH = os.path.join("..", "src", "data", "list_eval_partition.csv")
+IMAGES_PATH = os.path.join(
+    "..", "src", "data", "img_align_celeba", "img_align_celeba"
+)
```

Comment:
To nie są dobre miejsca do trzymania danych, bo to jest jakby wewnątrz repo. Polecam, żeby to było w configu jednak - pogadajmy o tym.